schemas/user.py

```
@@ -0,0 +1,61 @@
+#회원가입 요청
+class UserCreate(UserBase):
+    password: str = Field(...,min_length=8,max_length=20,description="비밀번호 (영문/숫자/특수문자 포함)")
+
+    @field_validator("password") # 클래스 내부의 필드 password
+    @classmethod # 아직 객체가 생성되지 않았으므로 클래스메서드 선언
+    def validate_password(cls,v: str) -> str: # 반환 타입 str
+        #re.search는 문자열 전체에서 해당 패턴이 하나라도 있는지 찾는다.
+
+        #영문 대문자 또는 소문자가 하나라도 포함되는지 확인
+        if not re.search(r"[A-Za-z]",v):
+            raise ValueError("비밀번호에 영문자가 포함되어야 합니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)